app/src/main/res/layout/fragment_onboarding_page.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do we need a scroll view here? I think if the text is going off the screen we should be shortening the text